FAQ/FAQ-Synchronization.md

```diff
@@ -125,8 +125,12 @@ To reindex your node:
 
 ```bash
 btcpay-down.sh
-rm -rf /var/lib/docker/volumes/generated_bitcoin_datadir/_data
-mkdir /var/lib/docker/volumes/generated_bitcoin_datadir/_data
+# Delete 'blocks' and 'chainstate' folders
+rm -rf /var/lib/docker/volumes/generated_bitcoin_datadir/_data/blocks
+rm -rf /var/lib/docker/volumes/generated_bitcoin_datadir/_data/chainstate
+# Recreate empty 'blocks' and 'chainstate' folders
+mkdir /var/lib/docker/volumes/generated_bitcoin_datadir/_data/blocks
```

Comment:
I don't think you need to recreate these dirs

Reply:
I think so too, yes. 
I kept it since @NicolasDorier recreated the '_data' folder as well in the commands before. He mentioned something to me about docker having issues with restarting with missing folders.

Reply:
Just checked again: Folders do definitely not need to be recreated. I'll remove these commands and commit it to this PR.